### ISSUE TITLE
Usage of `@providesModule`

### ISSUE THREAD

**Issue by FiberJW** [Kureev/react-native-list#1]
Hey there,

This is really useful for people using prior versions, but using `@providesModule` makes this unusable for Expo folks because `rn-cli.config.js` doesn't exist with Expo. With Expo we have to wait for them to publish a new sdk before we can use the latest RN version, so changing your code to use relative imports instead of `@providesModule` would make it much easier for users to get access to FlatList+. Your code is in the same directory so you don't really need that special module resolution. Just throw a `./` in your requires and we all prosper. 

Thanks for your work!

**Comment by Kureev** [comment]
Indeed! I'll try to improve it tomorrow. Thanks for the issue!

**Comment by FiberJW** [comment]
Update: Expo can use rn-cli.config.js, but it would still be nice to not have to add config

**Comment by Kureev** [comment]
Yeah, I'd like to make installation of this library as easy as possible, and `rn-cli.config.js` is definitely an unnecessary complication we can avoid. I'm on it right now, will let you know ASAP.

**Comment by Kureev** [comment]
Looks a bit more complicated than expected. I'll keep working on this next week.
